targets/mgmarkdownfmt/targets.go

```diff
@@ -0,0 +1,42 @@
+package mgmarkdownfmt
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/go-logr/logr"
+	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/sh"
+	"go.einride.tech/mage-tools/mglog"
+	"go.einride.tech/mage-tools/mgtool"
+)
+
+const version = "75134924a9fd3335f76a9709314c5f5cef4d6ddc"
```

Comment:
Why are we depending on a commit rather then a version?

Reply:
No recent version has been released